controllers/clusterresources/postgresqluser_controller.go

```diff
@@ -0,0 +1,340 @@
+) error {
+	parts := strings.SplitN(defaultUserNamespacedName, string(types.Separator), 2)
+	if len(parts) != 2 {
+		return fmt.Errorf("cannot SplitN slices %s into substrings for username: %s", defaultUserNamespacedName, newUserName)
+	}
+
+	defaultUserSecret := &k8sCore.Secret{}
+	userSecretNamespacedName := types.NamespacedName{
+		Name:      parts[1],
+		Namespace: parts[0],
+	}
+	err := r.Get(context.TODO(), userSecretNamespacedName, defaultUserSecret)
+	if err != nil {
+		if k8sErrors.IsNotFound(err) {
+			return fmt.Errorf("cannot get default PostgreSQL user secret, username: %s , err: %w", parts[1], err)
+		}
+
+		return err
+	}
+
+	defaultUsername, defaultPassword, err := getUserCreds(defaultUserSecret)
+	if err != nil {
+		return fmt.Errorf("cannot get default PostgreSQL user credentials, secret name: %s, err: %w", defaultUserSecret.Name, err)
+	}
+
+	clusterName := defaultUserSecret.Labels[models.ControlledByLabel]
+	clusterID := defaultUserSecret.Labels[models.ClusterIDLabel]
+
+	exposeServiceList, err := exposeservice.GetExposeService(r.Client, clusterName, parts[0])
+	if err != nil {
+		return fmt.Errorf("cannot list expose services for cluster: %s, err: %w", parts[0], err)
+	}
+
+	if len(exposeServiceList.Items) == 0 {
+		return models.ErrExposeServiceNotCreatedYet
+	}
+
+	exposeServiceEndpoints, err := exposeservice.GetExposeServiceEndpoints(r.Client, clusterName, parts[0])
+	if err != nil {
+		return fmt.Errorf("cannot list expose service endpoints for cluster: %s, err: %w", parts[1], err)
+	}
+
+	if len(exposeServiceEndpoints.Items) == 0 {
+		return models.ErrExposeServiceEndpointsNotCreatedYet
+	}
+
+	nodeList := &k8sCore.NodeList{}
+
+	err = r.List(context.TODO(), nodeList, &client.ListOptions{})
+	if err != nil {
+		return fmt.Errorf("cannot list nodes, err: %w", err)
+	}
+
+	// TODO: Handle scenario if there are no nodes with external IP
+
+	for _, node := range nodeList.Items {
+		for _, nodeAddress := range node.Status.Addresses {
+			if nodeAddress.Type == k8sCore.NodeExternalIP {
+				err := r.createPostgreSQLFirewallRule(node.Name, clusterID, parts[0], nodeAddress.Address)
+				if err != nil {
+					return fmt.Errorf("cannot create postgreSQL firewall rule, err: %w", err)
+				}
+			}
+		}
+	}
+
+	serviceName := fmt.Sprintf(models.ExposeServiceNameTemplate, clusterName)
+	host := fmt.Sprintf("%s.%s", serviceName, exposeServiceList.Items[0].Namespace)
+
+	dbURL := fmt.Sprintf("postgres://%s:%s@%s:%d/%s?target_session_attrs=read-write",
+		defaultUsername, defaultPassword, host, models.DefaultPgDbPortValue, models.DefaultPgDbNameValue)
+
+	conn, err := pgx.Connect(context.Background(), dbURL)
+	if err != nil {
+		return fmt.Errorf("cannot establish a connection with a PostgreSQL server, err: %w", err)
+	}
+	defer conn.Close(context.Background())
+
+	createUserQuery := fmt.Sprintf(`CREATE USER "%s" WITH PASSWORD '%s'`, newUserName, newPassword)
+	_, err = conn.Exec(context.Background(), createUserQuery)
```

Comment:
We cannot do this because it is important to have " " and ' ' in query

Reply:
Uhm, I didn't get what you meant. I gave the example with those '' in the query.

Reply:
I mean that example that you provided won't work. In the comment above, I described why

Reply:
Oh, I see. You have to use the quotes for the SQL.
 If you put those `" "` and `' '` quotes to the example of mine, does the problem still occur? Because it looks a bit strange, so I'm asking about it.

Reply:
I already tested your solution and it does not work